### PR TITLE
Fixed the problem of folder Renewal delay display abnormality.

### PR DESCRIPTION
**Pull request by FengZh61**
image

Whether you create or edit a folder, Renewal delay it always shows 0 after saving.

### CODE REVIEW

**Comment by FengZh61** [comment]
image
Fix Only display folders to depth duplicate display issue when switching roles multiple times in Roles page.

File: pages/roles.js.php [hunk 1]
```diff
@@ -257,7 +257,7 @@ function(data) {
                     $('.infotip').tooltip();
 
                     // Adapt select
-                    $('#folders-depth').val('').change();
+                    $('#folders-depth').val('').empty();
```
Comment (corentin-soriano) [inline pages/roles.js.php:260]: `empty()` should instead replace `val('')` while retaining `change()`

```javascript
$('#folders-depth').empty().change();
```

`empty()` will clear all old options and `change()` optionally trigger event listeners.

**Review by corentin-soriano — CHANGES_REQUESTED** [review]
You must also sign your commits.

To configure signature: https://docs.github.com/fr/authentication/managing-commit-signature-verification/about-commit-signature-verification

Re-sign your commits/correct the author:
```bash
git rebase -i master
# Replace 'pick' by 'edit' on each commit, save and exit.

# And for each commit
git commit --amend --reset-author -S
git rebase --continue

# Update remote branch:
git push --force
```

**Review by nilsteampassnet — APPROVED** [review]

**Comment by FengZh61** [comment]
> You must also sign your commits.
> 
> To configure signature: https://docs.github.com/fr/authentication/managing-commit-signature-verification/about-commit-signature-verification
> 
> Re-sign your commits/correct the author:
> 
> ```shell
> git rebase -i master
> # Replace 'pick' by 'edit' on each commit, save and exit.
> 
> # And for each commit
> git commit --amend --reset-author -S
> git rebase --continue
> 
> # Update remote branch:
> git push --force
> ```

First time using PULL, a little rusty XD

**Review by nilsteampassnet — APPROVED** [review]

**Review by nilsteampassnet — APPROVED** [review]

**Comment by nilsteampassnet** [comment]
I'm closing it without merge but I merged manually.

**Pull request closed without merging**